source/shared/core_stmt.cpp

```diff
@@ -422,6 +422,12 @@ void core_sqlsrv_bind_param( _Inout_ sqlsrv_stmt* stmt, _In_ SQLUSMALLINT param_
 
     // if it's an output parameter and the user asks for a certain type, we have to convert the zval to that type so
     // when the buffer is filled, the type is correct
```

Comment:
I think you should amend this comment to explain the error check that you added below. "But first, check that we are not using a LOB type with an output parameter" or something like that.